platform/tron/api.go

```diff
@@ -38,13 +38,23 @@ func Normalize(srcTx *Tx) (tx models.Tx, ok bool) {
 	switch contract.Parameter.(type) {
 	case TransferContract:
 		transfer := contract.Parameter.(TransferContract)
+
+		from, err := HexToAddress(transfer.Value.OwnerAddress)
```

Comment:
I would rather call this as `HexToBase58`

Reply:
How about `HexToBase58Check`? It isn't just standard base58 because it also contains a checksum.

Reply:
`HexToBase58Check` make sense, I think in it's commonly used as base58 or base58 address